wcfsetup/install/files/lib/system/template/plugin/TimeModifierTemplatePlugin.class.php

```diff
@@ -33,6 +33,11 @@ public function execute($tagArgs, TemplateEngine $tplObj)
         $isFutureDate = ($timestamp > TIME_NOW);
         $dateTime = DateUtil::getRelativeTime($dateTimeObject, $timestamp, $date, $time, $isFutureDate);
 
+        return \sprintf(
+            '<woltlab-core-time date="%s"></woltlab-core-time>',
```

Comment:
Can we render *some* placeholder as the `textContent` here, so that the times at least show up without JavaScript? Or will this cause FOUC?

Reply:
I’ve added a placeholder date. FOUC isn’t a concern, because this is an eager web component and thus is being executed in a micro task while the DOM is being parsed.